.github/workflows/deploy.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     - name: install
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17
+        go-version: 1.18
 
```

Comment:
@everettraven I found another place where we need to update the go version:

operator-sdk/images/operator-sdk/Dockerfile
> ENV GO_VERSION 1.16.4